mapswipe_workers/mapswipe_workers/generate_stats/generate_stats.py

```diff
@@ -0,0 +1,46 @@
+from mapswipe_workers.definitions import logger
+from mapswipe_workers.definitions import DATA_PATH
+
+from mapswipe_workers.generate_stats import project_stats
+from mapswipe_workers.generate_stats import overall_stats
```

Comment:
See my other note, you can merge these `import` lines into a single one, it's easier to read :)